packages/react/src/components/Tooltip/Tooltip.mdx

```diff
@@ -7,9 +7,30 @@ import * as TooltipStories from './Tooltip.stories.tsx';
 
 # Tooltip
 
+`Tooltip` brukes for å vise korte forklaringer til brukeren. 
+
 <Primary />
 <Controls />
 
+## Slik bruker du `Tooltip`
+
+Du kan enten sende inn en string eller et element som `children`. Dersom du sender inn et element, vil det elementet rendres i tooltipen.
+Dersom du bruker kun en string, vil `Tooltip` rendre en `span`.
```

Comment:
Some formulation help here is appreciated

Reply:
How about this? Maybe this is enough :D 
```
Du kan enten sende inn en `string` eller et element som `children`. Dersom du sender inn en `string`, vil denne legges i en `<span>`.
```

Reply:
Yes looks and sound good, traight to the point